demo/demo_director.py

```diff
@@ -936,6 +958,70 @@ def restore_timestamp(vin):
 
 
 
+def prepare_replay_attack_nokeys(vin):
+  """
+  For exposure via XMLRPC to web frontend, attack script to prepare to execute a
+  rollback attack with no compromised keys against the Director.
+
+  After this is done, the Primary should update, then the replay attack function
+  should be run (replay_attack_nokeys).
```

Comment:
I think it would be a good idea to mention what the preparation actually is/does -- `backup_timestamp`, `write_to_live`?